### PR TITLE
FS-4389: HSRA seed data config

### PR DESCRIPTION
**Pull request by hamzabinkhalid**
https://dluhcdigital.atlassian.net/browse/FS-4389

### Changes

- HSRA added in the seed data script
- Application form json created and added in seed_data

### CODE REVIEW

File: tests/test_seed_db.py [hunk 1]
```diff
@@ -55,7 +57,7 @@ def test_seed_application_completed(fund_config, round_config, _db, clear_test_d
 
 
 @pytest.mark.skip(reason="Needs running fund-store")
-@pytest.mark.parametrize("fund_config, round_config", [(COF, R3W1), (NSTF, R2)])
```
Comment (nuwan-samarasinghe) [inline tests/test_seed_db.py:58]: is this a known change ?

File: requirements.txt [hunk 2]
```diff
@@ -93,10 +91,8 @@ flask-sqlalchemy==3.0.3
     #   funding-service-design-utils
 flipper-client==1.3.1
     # via funding-service-design-utils
-funding-service-design-utils==2.0.41
+funding-service-design-utils==2.0.47
     # via -r requirements.in
-greenlet==3.0.1
```
Comment (nuwan-samarasinghe) [inline requirements.txt:98]: I guess we are removing some unwanted dependencies in here

**Review by nuwan-samarasinghe — APPROVED** [review]

File: tests/test_seed_db.py (hunk 1 above)
Comment (hamzabinkhalid) [reply tests/test_seed_db.py:58]: Yes

File: requirements.txt (hunk 2 above)
Comment (hamzabinkhalid) [reply requirements.txt:98]: It happens automatically when we run precompile command

**Pull request merged**